docs/source/configuration.rst

```diff
@@ -80,10 +83,61 @@ For example, a snippet from a :code:`pyproject.toml` file:
 .. _`cfg file`: https://docs.python.org/3/library/configparser.html
 .. _`pyproject.toml file`: https://www.python.org/dev/peps/pep-0518/
 
+
+.. _starter_config:
+
+New Project Configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When setting up a new project with SQLFluff, we recommend keeping your configuration
+file fairly minimal. The config file should act as a form of *documentation* for your
+team i.e. a record of what decisions you've made which govern how your format your
+SQL. By having a more concise config file, and only defining config settings
+where they differ from the defaults - you are more clearly stating to your team what
+choices you've made.
+
+*However*, there are also a few places where the *default* configuration is designed
+more for *existing projects*, rather than *fresh projects*, and so there is an
+opportunity to be a little stricter than you might otherwise be with an existing
+codebase.
+
+Here is a simple configuration file which would be suitable for a starter project:
+
+.. code-block:: cfg
+
+    [sqlfluff]
+    templater = jinja
+    dialect = override_this_with_your_project_dialect
+    # The standard max_line_length is 80 in line with the convention of
+    # other tools and several style guides. Many projects however prefer
+    # something a little longer.
+    max_line_length = 120
+
+    [sqlfluff:indentation]
+    # While implicit indents are not enabled by default. Many of the
+    # SQLFluff maintainers do use them in their projects.
+    allow_implicit_indents = true
+
+    # The default configuration for capitalisation rules is "consistent"
+    # which will auto-detect the setting from the rest of the file. This
+    # is less desirable in a new project and you may find this (slightly
+    # more strict) setting more useful. 
+    [sqlfluff:rules:capitalisation.keywords]
+    capitalisation_policy = lower
```

Comment:
Do you think everything being `lower` is the most common way? I'd think upper for keywords and types is more common (at least everywhere i've worked).

Reply:
I think it lower is indexed preferred because these days we have syntax highlighting with pretty colors so it’s not strictly necessary. 

https://stackoverflow.com/questions/608196/why-should-i-capitalize-my-sql-keywords-is-there-a-good-reason